### PR TITLE
fix(xmljs): correct DT owner GitHub handle

### PR DESCRIPTION
**Pull request by peterblazejewicz**
This corrects what looks like obvious omission.

/cc @blackshadev

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
Note, this requires explicit 'OK' from @blackshadev, before maintainer could merge this, thanks!

**Review by blackshadev — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
@orta, would you mind? Thx!

**Pull request merged**
